app/assets/stylesheets/master.css.scss

```diff
@@ -1,7 +1,7 @@
 td a {
-  display:block;
-  width:100%;
-  height: 100%;
+  display: block;
+  width: 100%;
+  height: 85%;
```

Comment:
i implemented @amarkpark's suggested fix here since i was staring at the pieces so much and it was starting to eat at my soul